src/util/browser.ts

```diff
@@ -407,6 +407,21 @@ export class Browser {
     this.browser.on("disconnected", () => {
       this.browser = null;
     });
+
+    // common permissions
+    const permissions = [
+      "notifications",
+      "geolocation",
+      "camera",
+      "microphone",
+    ];
+
+    for (const name of permissions) {
+      await this.firstCDP.send("Browser.setPermission", {
```

Comment:
nice addition!

Reply:
yes, i think this may help avoid some modal permission issues that arise on some sites.